release-team/role-handbooks/enhancements/README.md

```diff
@@ -75,43 +78,123 @@ It is important that this process be followed and documentation remain up-to-dat
+
+#### Dashboard
+
+The Dashboard tab is intended to be an _at-a-glance_ view of the current
+Enhancement status from both the perspective of the Enhancements and Docs teams.
+It is 100% generated from the `Enhancements` and `Docs` tabs and should **NOT**
+be updated manually.
+
+
+#### Enhancement Signal
+
+Enhancements that are missing any criteria should be labeled as `At Risk`. To
+help make this easier for the Enhancement team to label. Proposals and KEP State
+are color coded indicating their current readiness state.
+
+- `Proposal`
+  - ![#B7E0CD](https://placehold.it/12/B7E0CD?text=+) KEP is merged
+  - ![#FCE8B2](https://placehold.it/12/FCE8B2?text=+) KEP PR in flight
+  - ![#F4C7C3](https://placehold.it/12/F4C7C3?text=+) No KEP or KEP PR found
+- `KEP State`
+  - ![#B7E0CD](https://placehold.it/12/B7E0CD?text=+) `Implementable` or `Implemented`
+  - ![#FCE8B2](https://placehold.it/12/FCE8B2?text=+) `Provisional`
+  - ![#F4C7C3](https://placehold.it/12/F4C7C3?text=+) `none` or `invalid`
+
+
+#### Removing Untracked Enhancements
+
+Untracked Enhancements are Enhancements that were `Pending Inclusion` with either
+no response by the Enhancement Freeze deadline or specifically stated that they
+should not be included. Once either a response from issue owner stating it should
+not be tracked or the Enhancement Freeze deadline occurs, their status should be
+set to `Untracked`.
+
+Once done, use the `Enhancements` -> `Removed Untracked Enhancements` menu item
+to remove it from the `Dashboard`, `Enhancements` and `Docs` tabs. It will not
+be moved over to the `Removed from Milestone` tab.
+
+
+#### Removing Tracked Enhancements
+
+If the Enhancement is being bumped to a later release, set it's state to
+`Deferred`.
+
+If it is being removed due to missing criteria or lack of response
+after being included in the milestone, set its state to `Removed from Milestone`.
+
+Once done, use the `Enhancements` -> `Remove Enhancements from Milestone` menu
+item to automatically to move it to the `Removed from Milestone` tab removing it
+from the `Dashboard`, `Enhancements` and `Docs` tabs.
+
+
+#### Moving a Removed Enhancement Back into the Milestone
+
+If a removed item has had an exception granted. Set it's status to `Tracked` in
+the `Removed from Milestone` tab. Then use the `Enhancements` ->
+`Track Removed Enhancement` menu option to move it back to the `Dashboard`,
+`Enhancements`, and `Docs` tabs.
+
+
 ### Escalation / Handling Unresponsive Enhancement Owners
-For issues where the initial owner is unresponsive goin to the issue SIG leadership to check if the issue is still targeted for the release.
 
-If there is continue unresponsiveness on issues remove them from the milestone at your descretion.
+For issues where the initial owner is unresponsive going to the issue SIG leadership to check if the issue is still targeted for the release.
```

Comment:
"For issues where the initial owner is unresponsive, try escalating to the relevant SIG's leadership to determine if the issue is still target for the release."

Reply:
Updated 👍